zio-http/src/main/scala/zhttp/http/Header.scala

```diff
@@ -51,7 +51,7 @@ object Header {
     val authString    = String.format("%s:%s", username, password)
     val authCB        = Unpooled.wrappedBuffer(authString.getBytes(CharsetUtil.UTF_8))
     val encodedAuthCB = Base64.encode(authCB)
-    val value         = String.format("%s:%s", BasicSchemeName, encodedAuthCB.toString(CharsetUtil.UTF_8))
+    val value         = String.format("%s %s", BasicSchemeName, encodedAuthCB.toString(CharsetUtil.UTF_8))
```

Comment:
can add some unit tests on this.

Reply:
Tests already added for `basicHttpAuthorization`. Do we need to add test cases for string format?